boards/arm/arduino_nano_33_ble/arduino_nano_33_ble_defconfig

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52840_QIAA=y
+CONFIG_BOARD_ARDUINO_NANO_33_BLE=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
```

Comment:
Would you also enable HW_STACK_PROTECTION by default (it's now a recommendation for ARM boards)